### PR TITLE
HuggingFace examples, Python version change, Updated wheel file

### PR DESCRIPTION
**Pull request by NayantaraK**
- Added examples huggingface, huggingface-peft
- Updated the Python version supported
- Updated the wheel file

### CODE REVIEW

File: examples/huggingface-peft/README.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+Hugging Face PEFT
+=================
+
+This example runs a Hugging Face BERT movie reviews classification application with GPU based Parameter Efficient Fine-Tuning (PEFT) using LoRA (Low-Rank Adaptation) on the Swarm Learning platform.
```
Comment (suresh-ls) [inline examples/huggingface-peft/README.md]: This example runs a -> This example demonstrates the use of LLM fine-tuning in a Federated scenario. 

rest of the sentence is fine.

**Comment by NayantaraK** [comment]
**Huggingface Example Results (12k, 5 epochs)**

[INFO] Full Fine-tuning Evaluation Results:
==================================================
  eval_loss: 0.7146
  eval_accuracy: 0.7033
  eval_f1: 0.6882
  eval_precision: 0.6834
  eval_recall: 0.7033
  eval_runtime: 1.2723
  eval_samples_per_second: 943.1770
  eval_steps_per_second: 7.8600
  epoch: 4.9600
 
[INFO] Performance Summary:
==================================================
  Data Loading Time: 0.21 seconds
  Tokenization Time: 0.44 seconds
  Model Loading Time: 15.04 seconds
  Training Time: 462.07 seconds (7.70 minutes)
  Evaluation Time: 1.27 seconds
  Total Runtime: 491.12 seconds (8.19 minutes)
  Final Accuracy: 0.7033
  Final F1 Score: 0.6882
 
[INFO] Resource Usage Summary:
==================================================
  Total Parameters: 108,314,117
  Trainable Parameters: 108,314,117
  Model Size: ~413.2 MB (FP32)
  Sync Frequency: Every 16 batches
  Effective Batch Size: 128 samples
[INFO] Saving full fine-tuned model to /tmp/test/model/saved_models/n3_full/final
[INFO] Model saving completed in 0.28 seconds
[INFO] Full fine-tuned model saved successfully!


**Huggingface-peft Example Results (12k, 5 epochs)**

[INFO] Full Fine-tuning Evaluation Results:
==================================================
  eval_loss: 0.8497
  eval_accuracy: 0.6442
  eval_f1: 0.5896
  eval_precision: 0.5882
  eval_recall: 0.6442
  eval_runtime: 2.2793
  eval_samples_per_second: 526.4770
  eval_steps_per_second: 4.3870
  epoch: 4.9600
 
[INFO] Performance Summary:
==================================================
  Data Loading Time: 0.21 seconds
  Tokenization Time: 0.44 seconds
  Model Loading Time: 14.66 seconds
  Training Time: 440.37 seconds (7.34 minutes)
  Evaluation Time: 2.28 seconds
  Total Runtime: 470.09 seconds (7.83 minutes)
  Final Accuracy: 0.6442
  Final F1 Score: 0.5896
 
[INFO] Resource Usage Summary:
==================================================
  Total Parameters: 108,612,874
  Trainable Parameters: 298,757
  Model Size: ~414.3 MB (FP32)
  Sync Frequency: Every 16 batches
  Effective Batch Size: 128 samples
[INFO] Saving full fine-tuned model to /tmp/test/model/saved_models/n3_full/final
[INFO] Model saved in 0.59 seconds
[INFO] Total runtime: 470.68 seconds (7.84 minutes)
[INFO] PEFT model saved successfully!

**Review by suresh-ls — APPROVED** [review]
